app/forms/idv/in_person/address_form.rb

```diff
@@ -8,6 +8,12 @@ class AddressForm
 
       attr_accessor(*ATTRIBUTES)
 
+      attr_reader :capture_secondary_id_enabled
```

Comment:
WDYT of removing this and making a nice `?` method instead?

```suggestion
      def capture_secondary_id_enabled?
        @capture_secondary_id_enabled
      end
```

Reply:
Alternatively, we use this pattern in a few places to achieve the same effect:

https://github.com/18F/identity-idp/blob/fb439d84069cb49abef29ce4b113b330b99fa55b/app/presenters/session_timeout_modal_presenter.rb#L16-L17